testsuite/ocamlbuild_test.ml

```diff
@@ -32,10 +32,14 @@ let print_string_list_blank = print_list_blank pp_print_string
 
 let exists filename =
   try ignore(Unix.stat filename); true
-  with Unix.Unix_error ((Unix.ENOENT),_,_) -> false
+  with Unix.Unix_error ((Unix.ENOENT),_,_) ->
+    if Sys.win32
+    then Sys.file_exists filename
+    else false
```

Comment:
why not simply use `Sys.file_exists` on all platforms?
```suggestion
  Sys.file_exists filename
```

Reply:
I don't know how the behavior defer, I wanted to avoid changing the behavior in non windows as much as possible. But it's probably fine inside the testsuite.

Reply:
applied